README.md

```diff
@@ -1,6 +1,50 @@
 # SimpleITKPythonPackage
 
-This project provides a `setup.py` script that build SimpleITK Python wheel. [SimpleITK](http://www.simpleitk.org) is a simplified programming layer on top of the [Insight Segmentation and Registration Toolkit](https://itk.org) (ITK).  ITK is an open-source, cross-platform system that provides developers with an extensive suite of software tools for image analysis.
+This project provides a `setup.py` script that can build, install and package SimpleITK for Python. [SimpleITK](http://www.simpleitk.org) is a simplified programming layer on top of the [Insight Segmentation and Registration Toolkit](https://itk.org) (ITK).  ITK is an open-source, cross-platform system that provides developers with an extensive suite of software tools for image analysis.
+
+SimpleITK is available for binary downloads from [PyPI](https://pypi.python.org/pypi/SimpleITK) for many common platforms. Also a source distribution available of this repository which may be used when an appropriate binary [wheel](http://pythonwheels.com) is not available.
+
+To install SimpleITK:
+
+```bash
+pip install SimpleITK
+```
+
+## Installing SimpleITK for Python From the Python Packaging Source
+
+```bash
+pip install SimpleITK
+```
+
+### Prerequisites
+
+Building requires:
+* [CMake](https://cmake.org)
+* Git
+* C++ Compiler - Platform specific requirements are summarized in [scikit-build documentation](http://scikit-build.readthedocs.io).
+* Python
+** pip >= 9.0.0
+** setuptools >= 28.0.0
+
+Please ensure that `pip` and `setuptools` are up to date and are recent of enough version.
+
+### Compilations and Installation from Github
+
+SimpleITK can be compiled and install directly from the github repository:
+
+```bash
+pip install git+https://github.com/SimpleITK/SimpleITKPythonPackage.git
```

Comment:
I suggest to add `-v` .. the compilation being long, that will give some feedback.